### DISCUSSION TITLE
connecting devices and control in edge x

### DISCUSSION THREAD

**Discussion by Safinko** [edgexfoundry/community#103] [General]
Dear reader
(((EdgeX Foundry is an open source, vendor-neutral Edge IoT middleware platform that enables data exchange between IoT devices or sensors and related cloud or enterprise applications. The aim of the work is to verify the effectiveness of using EdgeX Foundry for the implementation of the peripheral part of the ICT / IoT network on a model example. Tasks: 1. Familiarize yourself with the EdgeX Foundry platform. 2. Design a sample service delivery solution using EdgeX Foundry. 3. Implement and evaluate the proposed solution. References: • [en.wikipedia.org/wiki/EdgeX_Foundry](http://en.wikipedia.org/wiki/EdgeX_Foundry) • [www.edgexfoundry.org](http://www.edgexfoundry.org/) • [github.com/edgexfoundry](http://github.com/edgexfoundry) • [ubuntu.com/internet-of-things/edgex](http://ubuntu.com/internet-of-things/edgex))) 
This is the text of my diploma
First point I have already done, and for second point I have prepared a project in Arduino which contains 2 Sensors one for Water detection and second one for water temperature and motor and fun by using ESP8266 Wifi and it is working but now my problem to connect this project with edge x foundary ((ESP8266 with MQTT server and Edgex))  I have tried a lot of resources but didn't help, I have installen edge X   Edinburgh 1.01 by this link https://jonamiki.com/2019/10/11/download-install-and-run-edgex-foundry-in-5-min-on-ubuntu-18-04-server/
I don't know if it is the best version but I couldn't install other one, which looks better ((because I have installed it on October last year and it was another a better version in the same port but now when I am trying it is new version and more complicated to use I think))  so can you help me please to connect this devices and control them by using edge x?  ,I have to submit this diploma on 16.05 and till now I am trying and not going. 😏
I will be so happy if you can help, can we make a call and I will explain more to you? 
Thank you
SAFI

**Comment by farshidtz** [comment]
To get help:
1. Explain the issue you are facing.
2. Choose a good title for your discussion.

The Edinburgh release is no longer supported. Supported versions right now are 2.1 (Jakarta) or 2.3 (Levski).

> I have to submit this diploma on 16.05 and till now I am trying and not going.

:)
